examples/resources/elasticstack_kibana_action_connector/resource.tf

```diff
@@ -10,3 +10,22 @@ resource "elasticstack_kibana_action_connector" "example" {
   })
   connector_type_id = ".index"
 }
+
+resource "elasticstack_kibana_action_connector" "pagerduty-connector" {
+  name = "pagerduty"
+  connector_type_id = ".pagerduty"
+  config = jsonencode({
+    apiUrl = "https://events.pagerduty.com/v2/enqueue"
+  })
+  secrets = jsonencode({
+    routingKey = pagerduty_service_integration.kibana.integration_key
+  })
+}
+.
```

Comment:
```suggestion

```

Reply:
corrected now. Please check. Also I am new to this contribution experience. I am not aware of GoLang as well. So could you please help me with what exactly is required as part of [appropriate integration test](https://github.com/elastic/terraform-provider-elasticstack/blob/main/internal/kibana/connector_test.go#L457)